### PR TITLE
update interactive examples in inline docstrings

### PR DESCRIPTION
**Pull request by weikang9009**
update interactive examples in the inline docstrings to comply with the api introduced in online docs

### CODE REVIEW

File: libpysal/weights/util.py [hunk 1]
```diff
@@ -1151,12 +1136,12 @@ def lat2SW(nrows=3, ncols=5, criterion="rook", row_st=False):
...
     --------
 
-    >>> from libpysal.weights import lat2W
+    >>> from libpysal.weights import lat2SW
     >>> w9 = lat2SW(3,3)
-    >>> w9[0,1]
-    1
-    >>> w9[3,6]
-    1
+    >>> w9[0,1] == 1
```
Comment (weikang9009) [inline libpysal/weights/util.py]: @jGaboardi this is working locally and on travis ci
If you can review this PR, and merge it if there is not any issue, the python version bumping PR should pass.

Comment (jGaboardi) [reply libpysal/weights/util.py]: So then I should remove the test adjustment in #154?

```
>>> w9[0,1]
array(1, dtype=int8)
>>> w9[3,6].tolist()
1
```

Comment (weikang9009) [reply libpysal/weights/util.py]: I think so. The test adjustment in #154 is not passing locally. I am not sure why...
But `numpy.array(1, dtype=numpy.int8)==1 ` actually awalys return `True`.

Comment (jGaboardi) [reply libpysal/weights/util.py]: hmmmm. The tests for #154 pass locally for me, both for Python `3.6` and `3.7.3` with `numpy 1.16.3`.

Comment (jGaboardi) [reply libpysal/weights/util.py]: Might it be an issue with your install?

Comment (weikang9009) [reply libpysal/weights/util.py]: Not sure. Maybe we can have a third person testing that.

Comment (jGaboardi) [inline libpysal/weights/util.py]: @weikang9009 and I are trying to figure out the relationship between the failing doctest (which I addressed in #154) and `numpy`. We need at least one more dev to weigh in here.

Comment (jGaboardi) [reply libpysal/weights/util.py]: @weikang9009 #154 has been merged into `master` by Serge.

Comment (jGaboardi) [reply libpysal/weights/util.py]: #155 reverting to pre-#154 until we get this issue worked out.

Comment (sjsrey) [reply libpysal/weights/util.py]: I had the same local failures as @weikang9009 so I don't think we have a good solution yet.

Comment (sjsrey) [reply libpysal/weights/util.py]: I think this is due to scipy going from 1.2.1 to 1.3.0. When I update to the latter I see
```
>>> w9[0,1]
array(1, dtype=int8)
```
but on 1.2.1 it is
```
>>> w9[0,1]
1
```

**Comment by jGaboardi** [comment]
#156

File: libpysal/weights/util.py (hunk 1 above)
Comment (weikang9009) [reply libpysal/weights/util.py]: Thanks for solving the mystery. I did not pay attention to the scipy version on my mac. Thought it was a numpy thing. Just noticed [`conda-forge` released `scipy 1.3.0`](https://github.com/conda-forge/scipy-feedstock/commit/d852089026f7b04110af5b67618110b4bef96a3a) and I did not update my local version.

**Pull request merged**